src/AuthProviders.ts

```diff
@@ -0,0 +1,8 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export enum AuthProviders {
+  Google = "google.com",
+  Email = "email"
+}
```

Comment:
Oh actually you know what, this is unused since we switched to using a custom token (JWT) generated by Firebase Auth. The original approach here was to actually log in with user credentials using UI, but we moved away from this.

Would you mind just removing this enum?

Reply:
Sure thing!